src/pl.x64.json

```diff
@@ -357,6 +365,7 @@
 			"folder-name": "GotoLineCol",
 			"display-name": "GotoLineCol",
 			"version": "2.2.0.2",
+			"npp-compatible-versions": "[8.0,]",
```

Comment:
I do understand the both "[,8.2.1]" and "[8.3.1,]". However what is the cause of the case "[8.0,]" ?

Reply:
FWDataViz and GotoLineCol depend on Dark Mode features, since [2.0.0.0] and [1.2.0.2], respectively; @shriprem has made this explicit in every recent release announcement since then:

> __*`ALERT`*__ This plugin release is only compatible with Notepad++ 8.0 and later releases.

Both were updated for wide `Sci_PositionCR` support [^1] [^2], but the pre-existing dependency remains, only with added nuance:
 
> __*`ALERT`*__ This plugin release is compatible with Notepad++ 8.0 or later releases. *However, release 8.3 or later is recommended.*

I don't read "recommended" as meaning "required". However, I also don't have any experience using either plugin, so I'm simply taking @shriprem's word for it.

[^1]: https://github.com/shriprem/FWDataViz/releases/tag/v2.4.0.2
[^2]: https://github.com/shriprem/Goto-Line-Col-NPP-Plugin/releases/tag/v2.2.0.2

[2.0.0.0]: https://github.com/shriprem/FWDataViz/releases/tag/v2.0.0.0
[1.2.0.2]: https://github.com/shriprem/Goto-Line-Col-NPP-Plugin/releases/tag/1.2.0.2

Reply:
@rdipardo 
Thank you for the detailed explanation.

Reply:
@rdipardo , I appreciate your diligent review of both my plugins' release notes, and your intent to accept them verbatim. In fact, your diligence caused me to _**actually**_ test the claims of version compatibility in those release notes.
1. GotoLineCol 2.2.0.2 worked equally well with Notepad++ versions 8.0, 8.3.0, 8.3.1 & 8.3.2. 
2.  FWDataViz 2.4.0.2 worked well with the newer Notepad++ versions 8.3.0, 8.3.1 & 8.3.2.  But this plugin release crashed the older Notepad++ 8.0 version while trying to use the plugin's [Data Extraction](https://github.com/shriprem/FWDataViz/blob/master/docs/data_extract_dialog.md) feature. Therefore I have now amended the [2.4.0.2 Release Notes](https://github.com/shriprem/FWDataViz/releases/tag/v2.4.0.2) to indicate that this plugin release is compatible with Notepad++ 8.3.1 or later versions.

After having to amend the [FWDataViz 2.4.0.2 Release Notes](https://github.com/shriprem/FWDataViz/releases/tag/v2.4.0.2), I went ahead and modified the [GotoLineCol 2.2.0.2 Release Notes](https://github.com/shriprem/Goto-Line-Col-NPP-Plugin/releases/tag/v2.2.0.2) as well with the same minimum compatibility of Notepad++ 8.3.1 version or later. There is no point in just this one plugin remaining an exception.

Please amend the compatibility metadata for both plugins to be: `[8.3.1,]`. Thank you in advance for making those changes.

Reply:
> Please amend the compatibility metadata for both plugins to be: `[8.3.1,]`. 

Done in rdipardo/nppPluginList@5e23fce